hail/python/hail/backend/service_backend.py

```diff
@@ -325,6 +325,49 @@ def render(self, ir):
         assert len(r.jirs) == 0
         return r(finalize_randomness(ir))
 
+    async def _submit_batch(self, build_batch_f, name, *, progress: Optional[BatchProgressBar] = None, **batch_args):
+        timings = Timings()
+
+        with timings.step("submit batch"):
+            token = batch_args.get('token') or secret_alnum_string()
+            batch_attributes = batch_args.get('attributes', {})
+            if 'name' not in batch_attributes:
+                batch_attributes = {**batch_attributes, 'name': self.name_prefix + name}
+            bb = self.async_bc.create_batch(token=token, attributes=batch_attributes)
+            build_batch_f(bb, self.flags)
+            b = await bb.submit(disable_progress_bar=True)
+
+        with timings.step("wait batch"):
+            try:
+                if self.disable_progress_bar is not True:
+                    deploy_config = get_deploy_config()
+                    url = deploy_config.external_url('batch', f'/batches/{b.id}/jobs/1')
+                    print(f'Submitted batch {b.id}, see {url}')
```

Comment:
Hmm, I think this should get nixed from here and `_rpc`. The new fancy progress bars already handle this really well. In fact, I never see this message printed which makes me think the if condition isn't right anyway.